charts/ndb-operator/values.yaml

```diff
@@ -17,7 +17,7 @@ serviceAccount:
   annotations: {}
   # The name of the service account to use.
   # If not set and create is true, a name is generated using the fullname template
-  name: "ndb-operator-service-account"
+  name: "ndb-operator-controller-manager"
```

Comment:
We should either let the user provide the values here, OR harcode these everywhere but **NOT** both. 
Note the following references to the service account:

1. charts/ndb-operator/templates/clusterrolebinding-ndb-operator-manager-rolebinding.yaml
2. charts/ndb-operator/templates/clusterrolebinding-ndb-operator-proxy-rolebinding.yaml
3. charts/ndb-operator/templates/serviceaccount-ndb-operator-controller-manager.yaml

These have a hardcoded service account name, whereas the other references are using the variable name. This will lead to inconsistencies and will break the functionality/installation. Please change to use variables throughout.

Reply:
Yes, discussed and sorted this out with @tuxtof. Was advised to hardcode it. We do not need to expose the service account name through templating logic as it is an internal entity and there is no use-case for the users to require a different service account name if it is unique enough eg ndb-operator-service-account.